<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:#FFFFFF;
}

QPushButton,QTextBrowser,QStackedWidget,QLineEdit
{
    border-style: solid;
    border: 2px solid silver;
    border-radius: 5px;
}
QPushButton:hover
{
    border-style: solid;
    border: 2px solid silver;
    border-radius: 5px;
    background-color: gainsboro;
}
QCheckBox:hover
{
    background-color: gainsboro;
}
QWight
{
    background-color: white;
}
QRadioButton
{
    border-style: solid;
    border: 2px solid silver;
    border-radius: 5px;
}
QComboBox
{
    border-style: solid;
    border: 2px solid silver;
    border-radius: 5px;
}
QComboBox:drop-down {
subcontrol-origin: padding;

subcontrol-position: top right;

width: 30px;

border-left-width: 0px;

border-left-color: gray;

border-left-style: solid;

border-top-right-radius: 3px;

border-bottom-right-radius: 3px;

}</string>
  </property>
  <widget class="QStackedWidget" name="Page">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>961</width>
     <height>601</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="1"/>
   </property>
   <property name="statusTip">
    <string comment="2"/>
   </property>
   <property name="whatsThis">
    <string comment="3"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget#Page
{
    border-style: solid;
    border: 2px solid silver;
    border-radius: 5px;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>180</width>
       <height>370</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>185</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>每日六战</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string extracomment="233"/>
       </property>
       <property name="text">
        <string>自动登录</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>人形制造</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>115</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>装备制造</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>45</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>自动后勤</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>动能模拟</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>255</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>势能模拟</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>325</y>
         <width>80</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>325</y>
         <width>80</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>反选</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>访问好友</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>装备强化</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5"/>
    </widget>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>六战后持续运行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>151</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>连接模拟器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>151</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>423</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>无动作</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>退出少女前线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>退出AS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>关闭模拟器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>退出并关闭模拟器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>休眠*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>关机*</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>425</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>完成后</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>指定次数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="running_text">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>490</width>
       <height>560</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser#running_text
{
    border-style: solid;
    border: 2px solid silver;
    border-radius: 5px;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_11">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>115</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>跳过强化</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>装备制造选择</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>光学瞄具</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>全息瞄具</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>红点瞄具</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>夜战装备</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>消音器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>穿甲弹</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>状态弹</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>高速弹</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>霰弹</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>外骨骼</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>防弹插板</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>弹链箱</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>伪装披风</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>备用瞄具</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>芯片</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>特殊穿甲弹</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>手枪</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>冲锋枪</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>步枪</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>突击步枪</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>机枪</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4442</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>人形制造选择</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>动能模拟选择</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>强化演习</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>资料采样</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>经验特训</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>经验特训-特种</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>云图回廊</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>防御演习</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>高级训练</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中级训练</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>初级训练</string>
      </property>
     </item>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'黑体'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>76</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>剩余动能:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>235</y>
       <width>50</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>76</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清除最新</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>721</width>
       <height>451</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_adb">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>20</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>adb路径(相对/绝对)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_select_adb">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>15</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_adb">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>15</y>
         <width>475</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>连接地址:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_address">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>475</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>431</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>请填写IP+端口  例如：127.0.0.1:62001</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>35</y>
       <width>149</width>
       <height>35</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>连接设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_debug">
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#Button_1
{
color:rgb(50, 108, 243);
border: 2px solid rgb(50, 108, 243);
border-radius: 0px;
border-style: hidden hidden solid hidden;
}
QPushButton#Button_1:hover
{
background-color: #FFFFFF ;
border-style: hidden hidden solid hidden;
}
QPushButton#Button_1:checked
{
border: 2px solid rgb(50, 108, 243);
border-radius: 0px;
border-style: hidden hidden solid hidden;
}</string>
   </property>
   <property name="text">
    <string>一键长草</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_2">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>0</y>
     <width>335</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#Button_2
{
border: 2px solid silver;
border-radius: 0px;
border-style: hidden hidden solid hidden;
}
QPushButton#Button_2:hover
{
background-color: #FFFFFF ;
border-style: hidden hidden solid hidden;
}
</string>
   </property>
   <property name="text">
    <string>任务配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>335</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#Button_3
{
border: 2px solid silver;
border-radius: 0px;
border-style: hidden hidden solid hidden;
}
QPushButton#Button_3:hover
{
background-color: #FFFFFF ;
border-style: hidden hidden solid hidden;
}
</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <zorder>Button_1</zorder>
  <zorder>Button_2</zorder>
  <zorder>Button_3</zorder>
  <zorder>Page</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
